playbook-swift/readME.md

```diff
@@ -0,0 +1,100 @@
+# Playbook for Swift 
+### Table of Contents
+	* [Folder Structure](bear://x-callback-url/open-note?id=05A5108D-03BF-4652-B3EE-33A2ABC7DF4C-497-0000E555E8E0C1FB&header=Folder%20Structure)
+	* [Package Manager](bear://x-callback-url/open-note?id=05A5108D-03BF-4652-B3EE-33A2ABC7DF4C-497-0000E555E8E0C1FB&header=Package%20Manager)
+	* [Release Process](bear://x-callback-url/open-note?id=05A5108D-03BF-4652-B3EE-33A2ABC7DF4C-497-0000E555E8E0C1FB&header=Release%20Process)
+	* [Extraction Plan](bear://x-callback-url/open-note?id=05A5108D-03BF-4652-B3EE-33A2ABC7DF4C-497-0000E555E8E0C1FB&header=Extraction%20Plan)
+
+
+---
+
+## Folder Structure
+The Playbook repo currently contains the following directories
+```
+/playbook
+/playbook-website
+```
+
+We will be following a monorepo approach when it comes to adding Swift UI Playbook Kits to this repo. The example below will be how our future folder structure will look like
+
+```
+/playbook
+/playbook-website
+/playbook-swift
+--/Sources/Playbook/
+--/Tests/PlaybookTests/
+--/package.swift
+/playbook-swift-app
```

Comment:
What is in this folder vs. what is in just the `/playbook-swift` folder? 
Do you have a spot where you got any inspiration from the naming convention structure you've created that I could look at to better understand this?

Reply:
With playbooks current folder structure we have a library folder next to a consumer folder for example 
/playbook
/playbook-website

I am just following the same convention we have established but with swift
/playbook-swift <---Library
/playbook-swift-app <--- Consumer this will behave like the website but will be a mobile app to demo our kits

@jasperfurniss